scripts/generators/beats.py

```diff
@@ -29,15 +29,21 @@
     FieldNestedEntry,
 )
 
+BEATS_DEFAULT_FIELDS = join(dirname(ecs_helpers.__file__), "beats_default_fields_allowlist.yml")
```

Comment:
If this file is intended to be customized, I think it would be better to add a generator.py argument that can be used to specify the file location used